pdr-admin/src/app/services/search.service.ts

```diff
@@ -0,0 +1,28 @@
+import { Injectable } from '@angular/core';
+import { DataService } from './data.service';
+import { Constants } from '../utils/constants';
+import { ApiService } from './api.service';
+
+@Injectable({
+  providedIn: 'root',
+})
+export class SearchService {
+  constructor(private dataService: DataService, private apiService: ApiService) {}
+
+  async fetchData(text) {
+    let queryParams = {};
+    if (text) {
+      queryParams['text'] = text;
+    }
+
+    // Calling API with status = null gives you current and historical
+    let res = await this.apiService.get('search', queryParams);
+    let data = this.apiService.getArrayFromSearchResults(res);
```

Comment:
both `const`

Reply:
fixed in [c37476f](https://github.com/bcgov/parks-data-register/pull/192/commits/c37476fa5ca9a4dc9435b661c605838b66b9b871)